src/api.js

```diff
@@ -0,0 +1,49 @@
+const list = document.querySelector('.scores');
+
+export const display = async () => {
+  await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/1WjHZNMPhymXrSt0JsIP/scores/',
+  )
+    .then((response) => response.json())
+    .then((json) => {
+      list.innerHTML = ' ';
+      for (let i = 0; i < json.result.length; i += 1) {
+        const item = document.createElement('li');
+        item.classList.add('users');
+        item.textContent = `${json.result[i].user} : ${json.result[i].score}`;
+        list.appendChild(item);
+      }
+    });
+};
+
+export const addList = async () => {
```

Comment:
- [ ] Your app is not running as expected, as submitted user data does not reflects on the scoreboard. Kindly ensure your data is persisted correctly.

Reply:
@rloterh can you please explain me briefly what is the error. 
I have checked everything and it is working, also related to scoreboard it is working and updating. I am sending you screenshot.  

![image](https://user-images.githubusercontent.com/105216647/186502991-3e6386d5-ad00-40a8-90c8-47faffc8a606.png) 

Update: 
![image](https://user-images.githubusercontent.com/105216647/186503070-b3b35370-fcf8-4b05-8635-86e5f709cdaa.png)